rbx_xml/src/types/numbers.rs

```diff
@@ -90,4 +139,22 @@ mod test {
             }
         );
     }
+
+    #[test]
+    fn test_inf_and_nan() {
+        test_util::test_xml_serialize::<Float32Type, _>(
+            r#"<float name="foo">INF</float>"#,
+            &std::f32::INFINITY,
+        );
+
+        test_util::test_xml_serialize::<Float32Type, _>(
+            r#"<float name="foo">-INF</float>"#,
+            &std::f32::NEG_INFINITY,
+        );
+
+        test_util::test_xml_serialize::<Float32Type, _>(
+            r#"<float name="foo">NAN</float>"#,
+            &std::f32::NAN,
+        );
+    }
```

Comment:
Can you add deserialization tests for these too?

Reply:
I'll work on it tomorrow.

Reply:
Done.